src/regressiontestoutput.txt

```diff
@@ -0,0 +1,23 @@
+
+FilePath: ../src/activities/testactivity/testactivity.js
+	 ERROR       Replace original GTK VERSION AUTHOR label by your own name
+	 WARNING     Replace original QT Quick copyright label email by your own email address
+
+FilePath: ../src/activities/testactivity/Testactivity.qml
+	 ERROR       Replace original GTK VERSION AUTHOR label by your own name
+	 WARNING     Replace original QT Quick copyright label email by your own email address
+	 WARNING     "testactivity activity" qsTr may not be used
+
+FilePath: ../src/activities/testactivity/ActivityInfo.qml
+	 WARNING     difficulty: 1 may not be updated.
+	 ERROR       author: "Your Name &lt;yy@zz.org&gt;" may not be updated.
+	 WARNING     "Your Name &lt;yy@zz.org&gt;" qsTr may not be used
+	 ERROR       demo: true may not be updated.
```

Comment:
this one should always be true actually (false is for activities on the paying version, and we won't add anymore paying activities).
It would be better to check the opposite.

Reply:
okay, didn't know that.